application/models/Question.php

```diff
@@ -774,15 +781,17 @@ private function sortAnswerOptions($answerOptions)
 
         // Sort alphabetically if applicable
         if ($this->shouldOrderAnswersAlphabetically()) {
+            if (is_null($language)) {
```

Comment:
did we need to check if language exist in survey->getAllLanguade ? Think it's already done in QuestionBaseRenderer ?

Reply:
Maybe double checking here wouldn't be the worst as to make things more atomic.
Just checking if the first answer is set would be enough, right?

Reply:
Ok